pywbem/_mof_compiler.py

```diff
@@ -2394,6 +2394,15 @@ def CreateInstance(self, *args, **kwargs):
                                 IncludeQualifiers=True)
             inst.path = CIMInstanceName.from_instance(
                 cls, inst, namespace=ns)
+
+        if "Abstract" in cls.qualifiers:
+            raise CIMError(
+                CIM_ERR_FAILED,
+                _format("Abstract class cannot be instantiated. Creation class "
+                        "{0!A} of new instance in Namespace {1!A} "
+                        "of the CIM repository has Abstract qualifier.",
```

Comment:
Can we simplify this message text, e.g. to:
"Cannot instantiate abstract class {0!A} in namespace {1!A}"

Reply:
Done.